statistics/histogram.go

```diff
@@ -352,6 +352,14 @@ func ValueToString(value *types.Datum, idxCols int) (string, error) {
 	return str, nil
 }
 
+func (hg *Histogram) bucketToString(bktID, idxCols int) string {
+	upperVal, err := ValueToString(hg.GetUpper(bktID), idxCols)
+	terror.Log(errors.Trace(err))
+	lowerVal, err := ValueToString(hg.GetLower(bktID), idxCols)
+	terror.Log(errors.Trace(err))
+	return fmt.Sprintf("num: %d lower_bound: %s upper_bound: %s repeats: %d", hg.bucketCount(bktID), lowerVal, upperVal, hg.Buckets[bktID].Repeat)
```

Comment:
`\t` may be better than space?

Reply:
I think space is better we usually only output one at a time in the log, it could save more space.